test/functional-portable/corerp/noncloud/resources/recipe_bicep_test.go

```diff
@@ -397,7 +397,7 @@ func Test_BicepRecipe_LanguageFailure(t *testing.T) {
 					// Instead of hardcoding values, we'll validate that the rest of the message is correct.
 					{
 						Code:            "DeploymentFailed",
-						MessageContains: "At least one resource deployment operation failed. Please see the details for the specific operation that failed.",
+						MessageContains: "At least one resource deployment operation failed.",
```

Comment:
The new version: `"At least one resource deployment operation failed. Please list deployment operations for details. Please see https://aka.ms/arm-deployment-operations for usage details."`